docs/HARDWARE-VERIFICATION.md

```diff
@@ -0,0 +1,79 @@
+# Verifying EVE on edge nodes
+
+This is a specialized version of EVE-OS installer that installs EVE-OS on the
+underlying edge node and verifies its compatibility. It checks if the necessary
+drivers are present, tests a static and dynamic networking configurations,
+tests the available storage devices, and tests if EVE-OS can spawn a guest
+successfully. Verification is currently supported on x86 edge nodes.
+
+## Running the verification image from boot media
+
+Similar to EVE-OS installer image (see [deployment](DEPLOYMENT.md)), we:
+
+1. Produce a disk-based installer image (e.g., by running `make verification-raw`, which creates the raw image file for verification, similar to what `make installer-raw` does).
+2. Burn the resulting ```verification.raw``` image file onto a USB stick and inserting it into the edge node.
+3. Have the machine boot from USB from BIOS.
+
+To produce the verification.raw image, we can execute the command
+```docker run lfedge/eve-verification:<tag> verification_raw > verification.raw```.
+We must select the `<tag>` from `lfedge/eve-verification` dockerhub repository
```

Comment:
Should we mention the minimum EVE version that supports verification and for which `lfedge/eve-verification` is available on the dockerhub?

Reply:
It is still not working correctly, so I will fix it and update the documentation with the minimum version.